config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
You are right, but you also agree with me that putting *any* code that is not necessary for the actual functionality of the app onto production servers is a bad idea, right? Especially since most of the styleguide stuff is barely tested.